api/v1/db/oracledb/games-dao.js

```diff
@@ -135,5 +154,5 @@ const isValidDeveloper = async (developerId) => {
 };
 
 module.exports = {
-  getGames, getGameById, postGame, isValidDeveloper,
+  getGames, getGameById, postGame, isValidDeveloper, deleteGame,
```

Comment:
Each property should be on it's own line: https://github.com/osu-mist/style-guides/blob/master/JavaScript.md#properties-spanning-multiple-lines